lib/util/component.js

```diff
@@ -41,7 +41,8 @@ function isReactComponent(context, node) {
  * @param {ASTNode} node The AST node being checked.
  * @returns {Boolean} True the node is a component definition, false if not.
  */
-function isComponentDefinition(node) {
+function isComponentDefinition(context, node) {
```

Comment:
Since you've changed the signature of the function, you should update its uses.
- https://github.com/yannickcr/eslint-plugin-react/blob/master/lib/rules/display-name.js#L124
- https://github.com/yannickcr/eslint-plugin-react/blob/master/lib/rules/prop-types.js#L528